Orange/widgets/utils/plot/owplotgui.py

```diff
@@ -439,7 +439,8 @@ class OWPlotGUI:
     JITTER_SIZES = [0, 0.1, 0.5, 1, 2, 3, 4, 5, 7, 10]
 
 
-    def __init__(self, plot):
+    def __init__(self, master, plot):
+        self._master = master
```

Comment:
OWLinePlot crashes. We could (temporarily) use
```
    def __init__(self, master, plot=None):
        plot = plot or master
```
I don't like it, but I cannot think of any better solution since OWLinePlot is in another add-on. The hack works because the widget only uses OWPlotGUI for Zoom/Select toolbar